folly/DynamicConverter.h

```diff
@@ -368,6 +369,25 @@ struct DynamicConstructor<std::pair<A, B>, void> {
   }
 };
 
+// vector<bool>
+template<class C>
+struct DynamicConstructor<C, typename std::enable_if<
+    std::is_same<C, std::vector<bool>>::value>::type>
```

Comment:
Could this just pattern-match against `vector<bool>` directly?

```lang=c++
template <>
struct DynamicConstructor<std::vector<bool>, void> {
```

Would that compile?

Reply:
Good point -- thanks for mentioning it! Yep, we can indeed use explicit template specialization here, which is cleaner than SFINAE. Allows us to remove the SFINAE from the "other ranges" specialization too.